.gitignore

```diff
@@ -216,8 +216,8 @@ FakesAssemblies/
 **/*.Server/ModelManifest.xml
 _Pvt_Extensions
 
-# App_Data folder
-**/App_Data
+# Data folder
+**/data
 
 # RavenDB stuff

```

Comment:
Hmm, I don't think this is necessary anymore, now that I look at it.

Reply:
I don't think it was necessary before the port, do you? FeatureStore was under App_Data for a while now...